packages/create-flex-plugin/templates/js/package.json

```diff
@@ -3,20 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "bootstrap": "flex-plugin check-start",
-    "prebuild": "rimraf build && npm run bootstrap",
-    "build": "flex-plugin build",
-    "clear": "flex-plugin clear",
-    "predeploy": "npm run build",
-    "deploy": "flex-plugin deploy",
-    "eject": "flex-plugin eject",
-    "info": "flex-plugin info",
-    "postinstall": "npm run bootstrap",
-    "list": "flex-plugin list",
-    "remove": "flex-plugin remove",
-    "prestart": "npm run bootstrap",
-    "start": "flex-plugin start",
-    "test": "flex-plugin test --env=jsdom"
```

Comment:
we don't want the start/test/build scripts either? or those will be ran with the cli too? don't have much context 😂

Reply:
Yes, CLI is to be the only tool to use